src/index.css

```diff
@@ -6,6 +6,7 @@
   --color-saffron: #ff671f;
   --color-white: #ffffff;
   --color-green: #046a38;
+  --color-purple: #cfc7e9;
```

Comment:
Don't you think it should be --color-primary

Reply:
Done